packages/cli/src/s3-store.ts

```diff
@@ -94,7 +94,8 @@ export class S3Store implements IKVStore {
   async exists(key: string, useCaseName?: string): Promise<boolean> {
     const store = await this.#storeMap.get(useCaseName)
     try {
-      return typeof (await store.get(key).toString()) === 'string'
+      const value = await store.get(key)
+      return value !== undefined
```

Comment:
Maybe:

```suggestion
      return Boolean(value)
```

Reply:
Thanks, but not sure about this. `Boolean(value)` also returns `false` for `""`, `0`, etc., right? I can imagine that this general S3 storage class may want to store `0`s as values for something some time :) .

Reply:
True. That's why "maybe" :)

Reply:
@pawartur please refer to javascript documentation. The cases you mentioned returns false.